src/Symfony/Component/HttpFoundation/File/UploadedFile.php

```diff
@@ -63,6 +64,7 @@ class UploadedFile extends File
     public function __construct(string $path, string $originalName, string $mimeType = null, int $error = null, bool $test = false)
     {
         $this->originalName = $this->getName($originalName);
+        $this->originalPath = $originalName;
```

Comment:
just for normalizing a bit:
```suggestion
        $this->originalPath = strtr($originalName, '\\', '/');
```

Reply:
Is that Windows related? Otherwise I don't get why.

Reply:
could be browsers on windows sending this yes